internal/browser-features/Browser.ts

```diff
@@ -66,10 +67,12 @@ export abstract class Browser {
 	}
 
 	public getPrefix(): string {
-		return (
-			this.getVersionConsumer().get("p").asStringOrVoid() ??
-			this.getDefaultPrefix()
-		);
+		if (this.prefix == null) {
```

Comment:
We never actually produce `null` inside of Rome, it's always `undefined`. The only time we use `== null` is when dealing with `String.prototype.match` return values.

It's longer, but more explicit to do:

```suggestion
		if (this.prefix === undefined) {
```